src/routes/+page.svelte

```diff
@@ -523,35 +522,40 @@
         <my-list-animes class="relative mt-4 grid-cols-3 md:grid-cols-5 gap-3 md:mt-[1vw] grid md:gap-[1.25vw]">
             {#each my_list as anime}
                 <a
-                    href="/mal/{anime.id}"
+                    href="/mal/{anime.id}/episode/{anime.current_episode}"
                     class="unstyled col-span-1 relative flex flex-col gap-2 md:gap-[0.5vw]"
-                    use:tippy={{
-                        arrow: false,
-                        allowHTML: true,
-                        placement: "right-start",
-                        offset: [10, 10],
-                        animation: "scale",
-                        duration: [150, 10],
-                        appendTo: document.body,
-                        onTrigger: async (instance) => {
-                            const node = document.createElement("div");
-                            new MyListAnimeDetails({
-                                target: node,
-                                props: {
-                                    anime_name: anime.name,
-                                    anime_type: anime.type,
-                                    anime_genres: anime.genres,
-                                    anime_studios: anime.studios,
-                                    anime_episodes_count: anime.episodes_count,
-                                    anime_synopsis: anime.synopsis,
-                                }
-                            });
-
-                            instance.setContent(node);
-                        }
-                    }}
                 >
-                    <div class="relative">
+                    <div
+                        class="relative"
+                        use:tippy={{
+                            arrow: false,
+                            allowHTML: true,
+                            placement: "right",
+                            offset: [100, -100],
+                            animation: "scale",
+                            duration: [150, 10],
+                            interactive: true,
+                            appendTo: document.body,
+                            onTrigger: async (instance) => {
+                                const node = document.createElement("div");
```

Comment:
Let's make it different.

Also no `createElement`. Use `cloneElement`

Reply:
New elements gets created on every trigger... this is bad... right?
cause memory leakages?